code/controllers/subsystem/events.dm

```diff
@@ -64,21 +64,24 @@ SUBSYSTEM_DEF(events)
 	var/players_amt = get_active_player_count(alive_check = 1, afk_check = 1, human_check = 1)
 	// Only alive, non-AFK human players count towards this.
 
+	var/players_amt_total = get_active_player_count(alive_check = 0, afk_check = 1, human_check = 1)
```

Comment:
Perhaps the human check should be false?  I'm not sure exactly what this parameter does.